### ISSUE TITLE
Wireless Bridge support

### ISSUE THREAD

**Issue by deevus** [ntadmin/DGND3700v2#56]
Is there any way to get wireless bridging working? Wireless repeating requires disabling security which I don't want to do.

**Comment by Forest-Feighner** [comment]
I've been thing about working on wireless bridging as well. My thought was that it could just be set up via wl.

I just noticed your post as I stoped by to mention to ntadmin that I noticed some flashing option for wl. I don't currently have access to the router but it's on my todo list.

**Comment by ntadmin** [comment]
I’m pretty sure it should be possible, and interesting to hear about the options for flashing.

Sadly, I’m not in a position to do much to the router anymore but very happy for others to go for it. Please do add to the distribution, I’m really happy for people to fork it and add to it and I can merge back in and release.

**Comment by ntadmin** [comment]
Sorry about the accidental close and reopen there.

**Comment by Ivoz** [comment]
Yes, it's quite sad that I have this older router sitting here, but it's only capable of wireless bridging with WEP security, which may as well be open for all the security that gives nowadays.

**Comment by ntadmin** [comment]
Well, it does make a good wireless base if you can get a wireless to it!
